charts/microgateway/Chart.yaml

```diff
@@ -40,3 +40,12 @@ annotations:
     - name: Airlock Minikube Example
       url: https://github.com/ergon/airlock-minikube-example
   artifacthub.io/license: MIT
+dependencies:
+  - name: redis
```

Comment:
The dependency seems to produce a warning during installation: coalesce.go:203: warning: destination for repository is a table. Ignoring non-table value bitnami/redis